builder/bundle_control.go

```diff
@@ -831,53 +752,35 @@ func createBundleFile(bundle string, path string) error {
 }
 
 // EditBundles copies a list of bundles from upstream-bundles to local-bundles
-// (if they are not already there) or creates a blank template if they are new,
-// and launches an editor to edit them. Passing true for 'suppressEditor' will
-// suppress the launching of the editor (and just do the copy or create, if
-// needed), and 'add' will also add the bundles to the mix.
-func (b *Builder) EditBundles(bundles []string, suppressEditor bool, add bool, git bool) error {
+// (if they are not already there) or creates a blank template if they are new.
+// 'add' will also add the bundles to the mix.
+func (b *Builder) EditBundles(bundles []string, add bool, git bool) error {
```

Comment:
This name doesn't really make sense anymore, since we no longer offer edit capabilities. Maybe we can use a different name, like CopyBundles or something? /me is open to suggestions.